.github/actions/notification/action.yml

```
@@ -16,4 +16,4 @@ runs:
     - name: Slack Notification
       shell: bash
       run: |
-          curl -X POST -H 'Content-type: application/json' --data '{"text":" -> [ STATUS: ${{ job.status }} WORKFLOW_NAME: ${{ github.workflow }}  JOB: ${{ github.job }}  JOB_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }} ] "}' ${{ inputs.slack_webhook_url }}
\ No newline at end of file
+          curl -X POST -H 'Content-type: application/json' --data '{"text":"\n[ \n\tSTATUS: \"${{ job.status }}\" \n\tWORKFLOW_NAME: \"${{ github.workflow }}\" \n\tJOB: \"${{ github.job }}\" \n\tJOB_URL: \"${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}\" \n] "}' ${{ inputs.slack_webhook_url }}
```

Nitpick, maybe we can remove the `[]` encapsulating the text. 
```suggestion
          curl -X POST -H 'Content-type: application/json' --data '{"text":"\n\tSTATUS: \"${{ job.status }}\" \n\tWORKFLOW_NAME: \"${{ github.workflow }}\" \n\tJOB: \"${{ github.job }}\" \n\tJOB_URL: \"${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}\""}' ${{ inputs.slack_webhook_url }}
```

---

done !